tests/auxiliary/test_spec_version.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+
```

Comment:
Why is this file in the `auxiliary` directory, instead of the root folder `tests`?

Reply:
I dunno 🤷 
Because the module to test is also in `auxiliary`. I take it, the tests shouldn't be there?

Reply:
`test_output` is also in `tests` and not `tests/auxiliary`. Though I understand your reasoning, I would keep the `auxiliary` folder in `tests` for auxiliaries needed for the tests, e.g. SBOMs.